### PR TITLE
Parsing stderr and identifying the key parts of the error.

### PR DESCRIPTION
**Pull request by briancohan**
Basic interaction for using pytui from the command line

### CODE REVIEW

File: pytui/main.py [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+    args = sys.argv
+    args[0] = 'python'  # Replace pytui with 'python' to act as a runner
+    result = subprocess.call(args, stderr=subprocess.PIPE, shell=False)  # noqa: S603
 
-    parser = PythonParser(all_text)
-    error_message = parser.parse_error()
-    backend = StackOverflowFinder()
-    search_results = backend.search(error_message)
-    Display(error_message, search_results).start()
+    if result.stderr:
```
Comment (VagishVela) [inline pytui/main.py:41]: I'm getting this error:
```
(.venv) ➜  Perceptive-Porcupines-Code-Jam-8 git:(interface) pytui example/runs_with_error.py
0
1
2
Traceback (most recent call last):
  File "/Users/vagishvela/.pyenv/versions/3.9.5/bin/pytui", line 8, in <module>
    sys.exit(main())
  File "/Users/vagishvela/.pyenv/versions/3.9.5/lib/python3.9/site-packages/pytui/main.py", line 41, in main
    if result.stderr:
AttributeError: 'int' object has no attribute 'stderr'
```

if you `print(result)` it just returns a `1`. Maybe something is missing in the code or setup?

**Review by VagishVela — CHANGES_REQUESTED** [review]

**Comment by VagishVela** [comment]
This would close #3

File: pytui/main.py (hunk 1 above)
Comment (ethansocal) [reply pytui/main.py:41]: `subprocess.call()` returns the error code of running the script. Maybe doing `subprocess.run()` would be better. https://docs.python.org/3/library/subprocess.html#subprocess.run

Comment (VagishVela) [reply pytui/main.py:41]: Looks like this fixes the issue

**Review by VagishVela — APPROVED** [review]
LGTM!

**Pull request merged**